frontend/src/Dashboard/CrossingListPage/CrossingListHeader/CrossingListHeader.scss

```diff
@@ -0,0 +1,104 @@
+.CrossingListHeader.fullsize {
+  height: 65px;
+  background: #0C0333;
+  display: flex;
+}
+
+.CrossingListHeader.smallsize {
```

Comment:
I'm noticing here that we are stacking classnames. This makes sense for adding modifiers to a component or block. But, this increases the specificity required to overwrite classes. We also run the risk of unknowingly creating a `smallsize` or `fullsize` class in the future in a different context and bumping into the limitations of CSS' lack of namespacing.

This is why I'm in favor of the BEM architecture for CSS. But, I'm open minded to other approaches. 

The approach I would recommend would be to choose the default size (small or full, doesn't matter to me) and using BEM to modify the default kinda like I do here in this basic example in the header:
https://github.com/cityofaustin/ctxfloods/blob/95bc22cbb91353ea974c5ad4f256ccb745fc4629/frontend/src/Dashboard/Header/Header.scss#L42-L56

@ifsimicoded and I talked a bit about this today. She is diving into this next and perhaps can come up with some recommendations and/or refactors that move us towards BEM or an appropriate alternative. Let's bookmark this for a conversation IRL along with testing.

Reply:
Definitely agree we should figure work towards a cohesive solution. Looking forward to discussing in meatspace.